neurom/apps/morph_stats.py

```diff
@@ -112,7 +122,13 @@ def extract_dataframe(neurons, config):
     if 'neuron' in config:
         del config['neuron']
 
-    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    func = partial(_run_extract_stats, config=config)
+    if n_workers == 1:
```

Comment:
I would have set a limit on the number of worker

Reply:
and adding a warning for the user saying : 
nworker too big setting nworker to X

Reply:
with max_worker = os.cpu_count() ?

Reply:
could be yes

Reply:
I don't think it's NeuroM responsability to restrain the number of workers. In my mind, it's really up to the user to pick the correct number of workers.

Reply:
meahhhh.
This is a question of risks/benefits. 
To me there is a small benefits in having a number of worker > os.cpu_count() (except maybe in very specific cluster related analyses where the number of available processors can be different than os.cpu_count() and or in I/O bound analaysis) and but there are some risks to slowdown a normal analysis. 

This is up to you !

Reply:
I will cut the pear in 2 then and set a warning but not restrict.

Reply:
me and @wizmer are the only users of that, and we know our stuff!

Reply:
I am fine with that